packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -73,6 +75,25 @@ const cfg = (options?: Options): Config => {
     },
     theme: {
       extend: {
+        screens: {
+          "sm-mm": { max: px(defaultTokens.breakpointMediumMobile) },
+          "mm-lm": {
+            min: px(defaultTokens.breakpointMediumMobile),
+            max: px(defaultTokens.breakpointLargeMobile - 1),
+          },
+          "lm-tb": {
+            min: px(defaultTokens.breakpointLargeMobile),
+            max: px(defaultTokens.breakpointTablet - 1),
+          },
+          "tb-de": {
+            min: px(defaultTokens.breakpointTablet),
+            max: px(defaultTokens.breakpointDesktop - 1),
+          },
+          "de-ld": {
+            min: px(defaultTokens.breakpointDesktop),
+            max: px(defaultTokens.breakpointLargeDesktop - 1),
+          },
+        },
```

Comment:
Do these classes belong to the `components` preset instead of `foundation`? The rest of the classes I see in the `components` preset have a prefix with the component name, but in this case they haven't.

Reply:
Wouldn't they fit better [here](https://github.com/kiwicom/orbit/blob/feat/migrate-hide/packages/orbit-tailwind-preset/src/presets/foundation/theme/screens.ts)?

Reply:
Yes, it has no component prefix, but it's part of the component preset (I know that we merged them), we document only the foundation, so that's why it's under chore. Do you think we should move that to the foundation and allow these range mq? I feel like we should avoid it and use only a mobile-first approach and these range queries might be useful in components like Hide 🤔

Reply:
I don't know, to be honest. I just wanted to have a little bit more context on the difference between both presets. I guess that consumers can achieve the same behavior via the Hide component, so no need to document those classes, is this correct?

Reply:
Yep, for hiding the content they can use Hide and do not need these range classes. To me, it's two different things and till now Orbit provided only mobile-first media queries, without ranges. So I think we should not introduce that now, they can add it to their configs if they want, but I guess it's quite a rare use case when it's needed.